gateway/api/views/programs.py

```diff
@@ -167,7 +147,28 @@ def list(self, request):
         tracer = trace.get_tracer("gateway.tracer")
         ctx = TraceContextTextMapPropagator().extract(carrier=request.headers)
         with tracer.start_as_current_span("gateway.program.list", context=ctx):
-            serializer = self.get_serializer(self.get_queryset(), many=True)
+
+            author = self.request.user
+            type_filter = self.request.query_params.get("filter")
+
+            if type_filter == "serverless":
```

Comment:
@korgan00 let's comment about these values, Monday. I think we can use the `Enum` approach that we used in the last PR but how they don't accept strings if we don't use `StrEnum` we will need to compare them in a different way probably.

Reply:
@korgan00 I just solved it in [86d1c3a](https://github.com/Qiskit/qiskit-serverless/pull/1540/commits/86d1c3aed10666695f87c9cb92e091c4a4452ce0)

and I could find a way to setup a Enum with strings for Python 3.9 👍